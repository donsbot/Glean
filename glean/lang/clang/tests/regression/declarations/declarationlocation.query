query: |
    { N, L } where cxx1.DeclarationLocationName { _, N, L }
transform: [gensort, []]
