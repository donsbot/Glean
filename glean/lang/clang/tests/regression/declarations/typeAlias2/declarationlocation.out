[
  "@generated",
  {
    "key": {
      "tuplefield0": {
        "file": { "key": "test.cpp" },
        "lineBegin": 10,
        "columnBegin": 1,
        "lineEnd": 15,
        "columnEnd": 1
      },
      "tuplefield1": "Foo"
    }
  },
  {
    "key": {
      "tuplefield0": {
        "file": { "key": "test.cpp" },
        "lineBegin": 11,
        "columnBegin": 3,
        "lineEnd": 11,
        "columnEnd": 14
      },
      "tuplefield1": "T1"
    }
  },
  {
    "key": {
      "tuplefield0": {
        "file": { "key": "test.cpp" },
        "lineBegin": 12,
        "columnBegin": 3,
        "lineEnd": 14,
        "columnEnd": 3
      },
      "tuplefield1": "Bar"
    }
  },
  {
    "key": {
      "tuplefield0": {
        "file": { "key": "test.cpp" },
        "lineBegin": 13,
        "columnBegin": 5,
        "lineEnd": 13,
        "columnEnd": 16
      },
      "tuplefield1": "T2"
    }
  },
  {
    "key": {
      "tuplefield0": {
        "file": { "key": "test.cpp" },
        "lineBegin": 17,
        "columnBegin": 1,
        "lineEnd": 23,
        "columnEnd": 1
      },
      "tuplefield1": "Foo"
    }
  },
  {
    "key": {
      "tuplefield0": {
        "file": { "key": "test.cpp" },
        "lineBegin": 19,
        "columnBegin": 3,
        "lineEnd": 19,
        "columnEnd": 14
      },
      "tuplefield1": "T3"
    }
  },
  {
    "key": {
      "tuplefield0": {
        "file": { "key": "test.cpp" },
        "lineBegin": 20,
        "columnBegin": 3,
        "lineEnd": 22,
        "columnEnd": 3
      },
      "tuplefield1": "Bar"
    }
  },
  {
    "key": {
      "tuplefield0": {
        "file": { "key": "test.cpp" },
        "lineBegin": 21,
        "columnBegin": 5,
        "lineEnd": 21,
        "columnEnd": 16
      },
      "tuplefield1": "T4"
    }
  },
  {
    "key": {
      "tuplefield0": {
        "file": { "key": "test.cpp" },
        "lineBegin": 25,
        "columnBegin": 1,
        "lineEnd": 25,
        "columnEnd": 15
      },
      "tuplefield1": "t1"
    }
  },
  {
    "key": {
      "tuplefield0": {
        "file": { "key": "test.cpp" },
        "lineBegin": 26,
        "columnBegin": 1,
        "lineEnd": 26,
        "columnEnd": 22
      },
      "tuplefield1": "t2"
    }
  },
  {
    "key": {
      "tuplefield0": {
        "file": { "key": "test.cpp" },
        "lineBegin": 27,
        "columnBegin": 1,
        "lineEnd": 27,
        "columnEnd": 22
      },
      "tuplefield1": "t3"
    }
  },
  {
    "key": {
      "tuplefield0": {
        "file": { "key": "test.cpp" },
        "lineBegin": 28,
        "columnBegin": 1,
        "lineEnd": 28,
        "columnEnd": 27
      },
      "tuplefield1": "t4"
    }
  }
]